README.md

```diff
@@ -40,7 +41,10 @@ On desktop platforms the extension internally renders to it's own OpenGL Frame B
 
 ## Project setup
 
-In `game.project` file add dependency `https://github.com/defold/extension-screenrecorder/archive/master.zip`.
+In `game.project` file add dependency 
+
+	https://github.com/dotgears/extension-screenrecorder/archive/master.zip
```

Comment:
That isn't true after the merge.
The main repo is in `defold:extension-screenrecorder`

Reply:
sorry, I did intend to test on our repo, removed anyway.